app/views/chatroom/index.html.erb

```diff
@@ -0,0 +1,112 @@
+<h2 class="ui center aligned medium icon header">
+  <i class="soundcloud icon"></i>
+  <div class="content">Say something</div>
```

Comment:
Why div & not span?

Reply:
Span is an in-line element and I don't expect to have anything beside this here so went with div which is a block element, having its own line.

Reply:
semantically, a div is a block. It could be anything. but if you use span / p, its indicating its a text. Which is this. So, span / p would be better for this.

You can easily use CSS to change it to `display: block`

Reply:
Would be semantically better to wrap the group as below instead?
```html
<div>
  <i></i>
  <h2></h2>
</div>
```

Reply:
actually, yes. LOL.
I didn't noticed that